### ISSUE TITLE
Create revised landing page

### ISSUE THREAD

**Issue by WillieCubed** [ConnieML/Connie-RTC#73]
**As a website visitor for Connie, I should be able to see a product marketing page for Connie so I can learn more about the service.**

# Acceptance Criteria

- ~~The website has a clear call-to-action to contact the ASA about gaining access to Connie~~
- ~~The website has information about Connie's features~~
- ~~The website gives context about why Connie is important~~
- ~~The website describes different use cases and user flows for Connie~~
- ~~There is a button/link to sign into Connie for existing users~~
- ~~All landing pages are accessible from the connieconnect.com domain~~
- The landing page has a button that starts the sign-in flow

# Tasks

- [ ] Create mockups for new landing pages
- [ ] Implement mockups in code using Next.js

# Additional Info

We'll need to work with @DeeGoldstein to further refine the acceptance criteria and get a better sense of the information that should be presented on the page.

**Comment by michelleshx** [comment]
@WillieCubed is this related to modifying the existing site: https://www.connieconnect.com/

**Comment by WillieCubed** [comment]
Yes, we'll have to work with Chris about updating the DNS settings for connieconnect.com to point to whatever deployment we'd like to make.

**Comment by WillieCubed** [comment]
Update: the connieconnect.com website will now solely be a page to sell Connie, hosted on Squarespace. There will be a new (but simple) internal landing page shown to the user before sign-in.
